### ISSUE TITLE
Great improvement. Training code release?

### ISSUE THREAD

**Issue by OwenHua666** [intel-isl/MiDaS#27]
The new model generates a more accurate depthmap than its previous version. Can you share the change you made? Also, is there a timeline for releasing the training code and the training data processing pipeline?

**Comment by ranftlr** [comment]
You might want to have a look at the first version of the paper to get a sense of what changed (https://arxiv.org/pdf/1907.01341v1.pdf). In a nutshell the final improvement is a culmination of multiple things:

- Better encoder and associated higher capacity
- More robust loss
- Higher resolution training
- More datasets for training 
- Better automatic cleaning procedure for movie groundtruth (i.e. the WSVD protocol)

There are currently no plans for releasing the training pipeline. For data see #13

**Comment by OwenHua666** [comment]
Thanks!
